### DISCUSSION TITLE
I am creating 100,000 pages with ssg, but the build time is taking more than 30 minutes.

### DISCUSSION THREAD

**Discussion by rlaqudwls25** [vercel/next.js#60156] [Show and tell]
I need to build more pages in the future. How can I solve this?

I'm currently trying to use the build cache feature using Next 14 TurboRepo. Is this a good idea?

help me plz 😢

**Comment by ahn0min** [comment]
This is something I was concerned about as well...
 I tried to solve it using SSR, but the cold start problem is still not resolved in Vercel.

**Comment by jungmin86** [comment]
I have trouble with the similar problem...
I don't know what to do :(
